Buildings/Fluid/HeatExchangers/DXCoils/AirCooled/Examples/DXCooledDataCenter.mo

```diff
@@ -0,0 +1,297 @@
+within Buildings.Fluid.HeatExchangers.DXCoils.AirCooled.Examples;
+model DXCooledDataCenter
+  "Example that illustrates the use of Buildings.Fluid.HeatExchanger.DXCoil in a data center room"
+  extends Modelica.Icons.Example;
+  replaceable package Medium =
+      Buildings.Media.Air;
+
+  // Air temperatures at design conditions
+  parameter Modelica.SIunits.Temperature TASup_nominal = 273.15+18
```

Comment:
@YangyangFu should use `TASup_nominal = 291.15` rather than `TASup_nominal = 273.15+18`. If `TASup_nominal = 273.15+18` is used, when instantiate the model in other model, the parameter value will not show correctly -- it will show like `TASup_nominal = 273.15 + 18` degC.